official/vision/beta/projects/yolo/modeling/tests/test_CSPConnect.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
```

Comment:
For all file names, please follow aaa_bbb_ccc.py. No cap. Thanks

Reply:
Fixed in 37434ec